CHANGELOG.md

```diff
@@ -1,8 +1,20 @@
+# Changelog	
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
 # [248.1.0](https://github.com/forcedotcom/commerce-on-lightning/compare/v242.1.1...248.1.0) (2023-09-11)
 
 
 ### Features
 
+* Order Management is now supported through the plugin. See examples/som/README.md for more details!
+
+* Digital Experience Bundle in Experience Cloud is now supported which means orgs that are higher than API version 59 can be used.
```

Comment:
[nit]: Digital Experience Bundle in Experience Cloud is now supported which means orgs that are higher than API version 59 can be used to create stores using the plugin